School.js

```diff
@@ -29,7 +29,8 @@ class School {
    * @return {Student} Enrolled student
    */
   enrollStudent(className, student) {
-    // Your code here
+    let newStudent = new Student(student.name, student.age, student.city, student.grade)
+    this.classes[className]["students"].push(newStudent)
```

Comment:
You'd want to return `newStudent` here so you could pass it in your response in app.js back to the frontend.